go/vt/vtgate/planbuilder/abstract/operator.go

```diff
@@ -48,12 +49,21 @@ func getOperatorFromTableExpr(tableExpr sqlparser.TableExpr, semTable *semantics
 	case *sqlparser.AliasedTableExpr:
 		switch tbl := tableExpr.Expr.(type) {
 		case sqlparser.TableName:
-			qg := newQueryGraph()
 			tableID := semTable.TableSetFor(tableExpr)
 			tableInfo, err := semTable.TableInfoFor(tableID)
 			if err != nil {
 				return nil, err
 			}
+			if tableInfo.IsVindexTable() {
```

Comment:
why add a method to the interface when we are doing a type cast in the next line?